intranet/celery.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from celery import Celery
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'intranet.settings')
```

Comment:
I'm really nitpicking: double quotes instead of single quotes.

Reply:
I agree. I was copy/pasting from something else, so it slipped through. Will fix.